src/MICore/CommandFactories/gdb.cs

```diff
@@ -258,5 +263,17 @@ private static string TypeBySize(uint size)
             }
             return TargetArchitecture.Unknown;
         }
+
+        public override async Task Signal(string sig)
+        {
+            string command = String.Format("-interpreter-exec console \"signal {0}\"", sig);
+            await _debugger.CmdAsync(command, ResultClass.running);
```

Comment:
Is the result class actually running after sending a signal? I'd expect it to be break, at least for sigint and signtrap.